tox.ini

```diff
@@ -64,13 +64,13 @@ deps =
 ignore_errors = true
 commands =
     # Global Linting Pass
-    pylint src/probnum --disable="no-member,abstract-method,arguments-differ,arguments-renamed,redefined-builtin,redefined-outer-name,too-many-instance-attributes,too-many-arguments,too-many-locals,too-many-lines,too-many-statements,too-many-branches,too-complex,too-few-public-methods,protected-access,unnecessary-pass,unused-variable,unused-argument,attribute-defined-outside-init,no-else-return,no-else-raise,no-self-use,else-if-used,consider-using-from-import,duplicate-code,line-too-long,missing-module-docstring,missing-class-docstring,missing-function-docstring,missing-param-doc,missing-type-doc,missing-raises-doc,useless-param-doc,useless-type-doc,missing-return-type-doc" --jobs=0
```

Comment:
Could you please add this change ("Removing line-too-long from the global linting pass") to the PR description as well? This way, it is easier to trace things back if issues come up. :)

Reply:
Yep, okay